### ISSUE TITLE
terminal_printer - print_answer fails is the given answer doesn't have attributes

### ISSUE THREAD

**Issue by The-Kor** [The-Kor/www#50]
Accessing the given answer's attributes (with .items()) should be done only after a validation that the given answer does have attributes.
In case the answer doesnt have attributes (for example GitHub answer) no attributes should be printed.
This causes exception thrown in the new pull request (with GITHUB implementation)
